.env-example

```diff
@@ -6,6 +6,9 @@ export COOKIE_DOMAIN="localhost"
 export AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS=3600000
 export JWT_SECRET=mysecretblah
 export FRONTEND_URL='http://localhost:8081'
+export GITHUB_ORG_NAME="isomerpages"
```

Comment:
Where is this used?

Reply:
It's an existing env var - it just wasn't in env-example previously